.github/workflows/container-image.yml

```diff
@@ -14,9 +14,17 @@ jobs:
       contents: read
 
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
Since we are touching this line, would you mind updating `actions/checkout`? AFAIK v4 is now the most recent one.